test_data/scripts/drb_2yr_check.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+source_dir=/Users/jamesmcc/usgs/data/prms_nhm_applications/drb
+target_dir=/Users/jamesmcc/usgs/data/prms_nhm_applications/drb_2yr
+
+run_dir=/Users/jamesmcc/usgs/data/prms_nhm_applications/drb_2yr_check
+source_run_dir=$run_dir/run_drb
+target_run_dir=$run_dir/run_drb_2yr
+rm -rf $source_run_dir
+rm -rf $source_target_dir
+mkdir -p $source_run_dir
+mkdir -p $target_run_dir
+
+prms=/Users/jamesmcc/usgs/prmsNHMpy/prms5.2.1/bin/prms
+
+
+# The OG
+cd $source_run_dir
+cp $prms .
+for ii in control.default myparam.param sf_data *cbh
+do
+    cp $source_dir/$ii .
+done
+# change the start and end dates by line number
+# change the output option
+# osx specific sed
+sed -i '' "6s/.*/1979/" control.default
+sed -i '' "7s/.*/1/" control.default
+sed -i '' "16s/.*/1981/" control.default
+sed -i '' "17s/.*/1/" control.default
+sed -i '' "136s/.*/2/" control.default
+./prms control.default -set param_file ./myparam.param
+
+
+# The Subset
+cd $target_run_dir
+cp $prms .
+for ii in control.default myparam.param sf_data *cbh
+do
+    cp $source_dir/$ii .
+done
+# change the start and end dates by line number
+# change the output option
+# osx specific sed
+sed -i '' "6s/.*/1979/" control.default
+sed -i '' "7s/.*/1/" control.default
+sed -i '' "16s/.*/1981/" control.default
+sed -i '' "17s/.*/1/" control.default
+sed -i '' "136s/.*/2/" control.default
```

Comment:
@jdhughes-usgs Let's talk about what other output we should generate and include as reference and what prms version to use for that.